fixtures/flutter_app/build/frontend_server.d

```diff
@@ -0,0 +1 @@
+frontend_server.d: /Users/jacobr/flutter/bin/cache/artifacts/engine/darwin-x64/frontend_server.dart.snapshot
```

Comment:
can we remove this file? anything with your username in it seems suspicious 🕵️‍♀️

Reply:
removed this whole directory.